conduit/pipeline/pipeline.go

```diff
@@ -372,7 +386,7 @@ func (p *pipelineImpl) Init() error {
 }
 
 func (p *pipelineImpl) Stop() {
-	p.cf()
+	p.ccf(errStopCause)
```

Comment:
nit: rename the var? the message `"pipeline stopped"` doesn't seem like an error?

Reply:
I totally agree that this isn't an error situation. However, cancellation causes must be errors and the linter doesn't like me creating errors that I don't name with `err` prefix. Shall I tray to figure out how to get the linter to stop complaining?

Reply:
VS Code still complains, but it should be passing lint now: deec57e61df93b3293e03eae9871ae8464785418